jwst/cube_build/cube_build_step.py

```diff
@@ -584,24 +598,31 @@ def check_offset_file(self):
                 continue
             else:
                 af.close()
-                raise ValueError('Error in offset file. A file in the association is not found in offset list %s', file_check)
+                raise ValueError(
+                    "Error in offset file. A file in the association is not found in offset "
+                    " list %s",
+                    file_check,
+                )
```

Comment:
This is not formatted properly for an error message.  It should be an f-string:
```suggestion
                raise ValueError(
                    f"Error in offset file. A file in the association is not found in offset "
                    " list {file_check}"
                )
```